ground-core/pom.xml

```diff
@@ -200,6 +117,13 @@
             <version>1.0.0-1</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/jline/jline -->
```

Comment:
I thought we needed to remove Titan because it was pulling in the wrong version of JLine; why do we need to explicitly import it?